build-system-tests/templates/components/vue/nuxt/app.vue

```diff
@@ -1,9 +1,37 @@
+<template>
+  <authenticator>
+    <template v-slot="{ user, signOut }">
+      <h1>Hello {{ user.username }}!</h1>
+      <button @click="signOut">Sign Out</button>
+    </template>
+  </authenticator>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'App',
+});
+</script>
+
 <script setup lang="ts">
 import { Authenticator } from '@aws-amplify/ui-vue';
 import '@aws-amplify/ui-vue/styles.css';
+
+import { Amplify } from 'aws-amplify';
```

Comment:
Is it normal for the imports not to be at the top in vue files?

Reply:
I think it's not _abnormal_, looking at their [style guide](https://v2.vuejs.org/v2/style-guide/#Single-file-component-top-level-element-order-recommended) it seems like either template or script should come first and style should come last. But most of the "create vite app" frameworks put the `script` tag first so I can do that.

Reply:
Updated, here is new workflow run https://github.com/aws-amplify/amplify-ui/actions/runs/10721583152